rules/cross-platform/external_alerts.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an external alert is received. Enabling this rule allows you to immediately being
```

Comment:
```suggestion
Generates a detection alert each time an external alert is received. Enabling this rule allows you to immediately begin
```

Reply:
TODO: Also add that index will default to configured `securitySolution:defaultIndex`